roles/scaffold_rm_facts/templates/module_utils/network_os/utils/utils.py.j2

```diff
@@ -5,3 +5,39 @@
 # (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 # utils
+{% if transport.netconf %}
```

Comment:
This template is not required as the code is moved out to junos/utils folder to a more commonplace in upstream `network/common/netconf.py`